src/main.cpp

```diff
@@ -5909,7 +5908,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         pfrom->setAskFor.erase(inv.hash);
         mapAlreadyAskedFor.erase(inv);
 
-        if (!AlreadyHave(inv) && AcceptToMemoryPool(mempool, state, tx, true, &fMissingInputs))
+        if (!AlreadyHave(inv) && AcceptToMemoryPool(mempool, state, tx, false, &fMissingInputs))
```

Comment:
Do we need to pass the fLimitFree flag still? Or can that be removed from the `AcceptToMemoryPool` signature (i.e. are all the call sites now passing `false`?

Reply:
all the `AcceptToMemoryPool()` calls do seem to set `fLimitFree = false`. but there was a commit in bitcoin/bitcoin which set `fLimitFree = true` to remove free transactions (https://github.com/bitcoin/bitcoin/pull/8287/files). so we may want to keep it around for future use.